inc/Widget/SmailyWidget.php

```diff
@@ -147,54 +147,307 @@ public function widget( $args, $instance ) {
+					// Show selected autoresponder.
+					if ( ! empty( $instance['autoresponder'] ) ) {
+						$current_autoresponder = json_decode( $instance['autoresponder'], true );
+						echo '<option value="' . $instance['autoresponder'] . '">' . $current_autoresponder['name'] . '</option>';
+					}
+					// Show all autoresponders from Smaily.
+					if ( ! empty( $autoresponder_list ) && ! array_key_exists( 'empty', $autoresponder_list ) ) {
+						echo '<option value="">' . esc_html__( '-No Autoresponder-', 'smaily' ) . ' </option>';
+						foreach ( $autoresponder_list as $autoresponder ) {
+							echo '<option value="' . htmlentities( json_encode( $autoresponder ) ) . '">' . $autoresponder['name'] . '</option>';
+						}
+					}
+					?>
+				</select>
+				<p>
+					<?php esc_attr_e( 'Select a suitable form-submitted trigger automation workflow from the list or create a new autoresponder on your Smaily account to be added to the list.', 'smaily' ); ?>
+				</p>
+			</div>
+			<?php endif; ?>
+
+			<!-- Write layout outside of autoresponders, so it would be always displayed, regardless autoresponders are created or not. -->
+			<!-- Layouts. -->
+			<div class="section">
+				<h2>
+					<?php esc_attr_e( 'Layout', 'smaily' ); ?>
+				</h2>
+				<p>
+					<?php esc_attr_e( 'Choose a preferred layout.', 'smaily' ); ?>
+				</p>
+
+				<div class="smaily-layout-container">
+					<p> <!-- Make the selected layout radiobutton active. -->
+						<input class="radio" id="<?php echo $this->get_field_id( 'smaily-layout-1' ); ?>" name="<?php echo $this->get_field_name('smaily_layout'); ?>"
+						type="radio" value="smaily-layout-1" <?php if($smaily_layout === 'smaily-layout-1'){ echo 'checked="checked"'; } ?> />
+						<label for="<?php echo $this->get_field_id( 'smaily-layout-1' ); ?>">
+						<strong>Email (Layout 1)</strong></label>
+					</p>
+					<p>
+						<img src="<?php echo SMAILY_PLUGIN_URL . "static/layouts/email-layout1.svg"; ?>" alt="smaily layout 1" class="smaily-layouts">
+					</p>
+				</div>
+				<div class="smaily-layout-container">
+					<p>
+						<input class="radio" id="<?php echo $this->get_field_id( 'smaily-layout-2' ); ?>" name="<?php echo $this->get_field_name('smaily_layout'); ?>"
+						type="radio" value="smaily-layout-2" <?php if($smaily_layout === 'smaily-layout-2'){ echo 'checked="checked"'; } ?> />
+						<label for="<?php echo $this->get_field_id( 'smaily-layout-2' ); ?>">
+						<strong>Email (Layout 2)</strong></label>
+					</p>
+					<p>
+						<img src="<?php echo SMAILY_PLUGIN_URL . "static/layouts/email-layout2.svg"; ?>" alt="smaily layout 2" class="smaily-layouts">
+					</p>
+				</div>
+				<div class="smaily-layout-container">
+					<p>
+						<input class="radio" id="<?php echo $this->get_field_id( 'smaily-layout-3' ); ?>" name="<?php echo $this->get_field_name('smaily_layout'); ?>"
+						type="radio" value="smaily-layout-3" <?php if($smaily_layout === 'smaily-layout-3'){ echo 'checked="checked"'; } ?> />
```

Comment:
Thanks for making each input element use the same formatting logic, but I think in doing that you reduced the readability of the code.

I suggest you use:
```html
<input
	type="text"
	class="foobar"
	name="foo"
	value="baz" />
```
formatting on input elements, instead of:
```html
<input type="text" class="foobar"
name="foo" value="baz" />
```